src/pug/blog/report_23.pug

```diff
@@ -0,0 +1,94 @@
+
+  +figure(
+    '',
+    'RF test 1',
+    '/static/blog/RF_proto_1.jpg',
+    'small inline',
+  )
+  +figure(
+    '',
+    'RF test 2',
+    '/static/blog/RF_proto_2.jpg',
+    'small inline',
+  )
+  +figure(
+    '',
+    'Connectors test',
+    '/static/blog/RF_proto_2b.jpg',
+    'small inline',
+  )
+  +figure(
+    '',
+    'RF test 3',
+    '/static/blog/RF_proto_3.jpg',
+    'small inline',
+  )
+
+  h4 RF test 1 board
+  ul
+    li A breakout board with a standalone #[em Nordic NRF24] and a 2.4GHz chip antenna.
+    li An external microcontroller (or a Pico) uses the wireless solution via #[em SPI].
+    li End-to-end latency: #[em ~1 millisecond].
+    li Jitter: So small we cannot even measure it (possibly #[em microseconds] on average).
+    li Connection range: #[em Excellent], able to transmit even from the next room.
+
+  h4 RF test 2 board
+  ul
+    li A full board with #[em Raspberry RP2040] microcontroller, Nordic NRF24 and a 2.4GHz chip antenna.
+    li #[em USB-C], but because a design error it was unusable.
+    li #[em Snap-in connectors] so it can be attached to a main board without soldering.
+    li Castellated pins for UART.
+    li We realized that the USB-C receptacle was 0.2 millimeters too tall.
+
+  h4 Connector breakout test
+  ul
+    li A simple connector-to-headers breakout board that allowed us to test the connectors.
+    li Connectors have a very #[em satisfying snap-in click].
+    li Connectors have very solid communication even when only partially plugged.
+    li Physical connection is very robust but can move a bit if the USB-C cable is twisted. Would require additional 3D-printed support when attached to a full controller.
+
+  h4 RF test 3 board
+  ul
+    li USB-C works as expected!
+    li USB-C receptacle is sunk into the board for extra clearance.
+    li #[em RP2040], flash memory, crystal clocks, linear regulator, and all other boring components works as expected.
+    li Power bridge allows us to test battery charging with external components.
+    li Unfortunately there was a #[em regression in wireless connection quality], the range was greatly reduced, and the antennas became very susceptible to angle changes.
+
+  h4 Next steps
+  ul
+    li Introduce #[em Microchip MCP73871] battery charging chip that we tested previously (the space is very limited due to the connectors).
+    li Investigate how to bring back great signal performance of the first prototype / Which changes are to blame for the reduction in connection quality.
+    li Create an Alpakka board prototype with snap-in connectors instead of soldering pads.
+
+  hr
+
+  h3 Maria01 Pitching Day
+  p #[+ax('https://maria.io') Maria01] (the startup campus in which Input Labs is located) organizes #[+ax('https://maria.io/events/maria-365-pitching-day/') 365 Pitching Day], an event in which startups showcase their work, not to actual VC companies, but as a #[em fun exercise] / competition (with judges!).
```

Comment:
Well, I don't know for whom it was fun.
At least one of the judges was always a VC.
So, maybe something like "different kinds of judges, from the VC, media and tech domain."